source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -268,10 +268,13 @@ export class InputConfig {
     });
     readonly debounceTimeInMilliSeconds: number = 0;
 
+    allowRevealOnHoverStyle: boolean = true;
+
     constructor(obj?: any) {
         if (obj) {
             this.label = new InputLabelConfig(obj["label"]);
             this.errorMessage = new BaseTextDefinition(obj["errorMessage"]);
+            this.allowRevealOnHoverStyle = obj["allowRevealOnHoverStyle"] || this.allowRevealOnHoverStyle;
```

Comment:
Why is this an `or` statement?

Reply:
Is this meant to be an "opt-in" feature?

Reply:
why do you need 'or'?

Reply:
incase obj["allowRevealOnHoverStyle"] is null, it will resolve to false. I followed similar to allowCustomStyle 

export class AdaptiveCardConfig {
    allowCustomStyle: boolean = false;

    constructor(obj?: any) {
        if (obj) {
            this.allowCustomStyle = obj["allowCustomStyle"] || this.allowCustomStyle;
        }
    }
}

Reply:
Yes its an opt-in feature. revealOnHover is only defined for web/win32 experiences where mouse hover is there. Its not intended to be lighten up for mobile. There are apps like Outlook mobile which uses adaptive card js sdk inside webview. So, we dont want to enable this feature for them, as mobile view should directly show inputs as editable field as there is no hover experience.

Thus, adding this flag where host can choose to enable/disable it.

Reply:
Makes sense, thank you for explaining!